src/bootlint.js

```diff
@@ -36,6 +36,31 @@ var LocationIndex = _location.LocationIndex;
     var IN_NODE_JS = !!(cheerio.load);
     var MIN_JQUERY_VERSION = '1.9.1';// as of Bootstrap v3.3.0
     var CURRENT_BOOTSTRAP_VERSION = '3.3.2';
+    var BOOTSTRAP_VERSION_4 = '4.0.0';
+    var BOOTSTRAP_VERSION_5 = '5.0.0';
+    var PLUGINS = [
+            'affix',
```

Comment:
You're right. Sorry about that. Result of Sublime's odd copy / paste behavior. No excuse for a sloppy thing like that though.